indigo_app/static/javascript/indigo/views/document_sentences_caser.js

```diff
@@ -0,0 +1,43 @@
+(function(exports) {
+  "use strict";
+
+  if (!exports.Indigo) exports.Indigo = {};
+  Indigo = exports.Indigo;
+
+  /**
+   * Handle the sentence caser view.
+   */
+  Indigo.DocumentSentenceCaseView = Backbone.View.extend({
+    el: '#sentence-caser',
+    events: {
+      'click .sentence-case-headings': 'sentenceCaseHeadings'
+    },
+
+    sentenceCaseHeadings: function(e) {
+      var self = this,
+          $btn = this.$el.find('.sentence-case-headings'),
+          data = {'document': this.model.document.toJSON()};
+
+      data.document.content = this.model.toXml();
+
+      $btn
+        .prop('disabled', true)
+        .find('i').addClass('fa-spin');
```

Comment:
I'm not sure this does anything since it's not a real button and I don't think there's a spinner available as we're not in a modal?

Reply:
This can be removed, there's no place to put the spinner in the menu any way.